src/faucet/Erc20Minter.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.18;
+pragma solidity ^0.8.13;
```

Comment:
I was thinking whether we should lock the compiler version or not, I would suggest to do so if we are done with the development, as per the best practices: https://consensys.github.io/smart-contract-best-practices/development-recommendations/solidity-specific/locking-pragmas/

Reply:
Aside from the security implications, it can also help us with verifying the deployed bytecode.

Reply:
Ok yea will keep that in mind, but it was giving me compilation errors so I changed it. We'll see if we can lock version prior to mainnet launch